src/main/resources/site/pages/default/default.html

```diff
@@ -147,7 +147,7 @@ <h1 data-th-text="${page.displayName}"></h1>
       data-th-unless="${statbankWeb}"
     >
       <div data-th-if="${region.showGreyTriangle}" class="grey-triangle"></div>
-      <div data-th-classappend="${region.showGreyBackground ? 'region-grey' : (region.topBorder ? 'border-columns' : '')}">
+      <div data-th-classappend="${region.showDarkGreyBackground ? 'region-dark-grey' : region.showGreyBackground ? 'region-grey' : (region.topBorder ? 'border-columns' : '')}">
```

Comment:
Må kanskje ha parantes for valg etter region-dark-grey så vi er sikker på at den velger rett, slik 
<div data-th-classappend="${region.showDarkGreyBackground ? 'region-dark-grey' : (region.showGreyBackground ? 'region-grey' : (region.topBorder ? 'border-columns' : ''))}">